Deploy-CAM.ps1

```diff
@@ -2519,6 +2526,59 @@ function Confirm-ModuleVersion()
     return $true
 }
 
+# Create a custom role for CAM with necessary permissions
+# Currently it is the based off of the Contributor Role excpept we remove Storage Access and VM Disk Blob Read Access
+# Use 'Get-AzureRmProviderOperation *' to get a list of Azure Operations and their details
+# See https://docs.microsoft.com/en-us/azure/active-directory/role-based-access-built-in-roles for details on Azure Built in Roles
+function Get-CAMRoleDefinition() {
+    param(
+        [parameter(Mandatory = $true)]
+        [String]$subscriptionId
+    )
+
+    $camCustomRoleDefinition = Get-AzureRmRoleDefinition "Cloud Access Manager"
+    # Create Role Defintion Based off of Contributor if it doesn't already exist
+    if ( -not $camCustomRoleDefinition ) {
+        $camCustomRoleDefinition = Get-AzureRmRoleDefinition "Contributor"
+        $camCustomRoleDefinition.Id = $null
+        $camCustomRoleDefinition.IsCustom = $true
+        $camCustomRoleDefinition.Name = "Cloud Access Manager"
+        $camCustomRoleDefinition.Description = "Required Permissions for Cloud Access Manager"
+        $camCustomRoleDefinition.AssignableScopes.Clear()
+        $camCustomRoleDefinition.AssignableScopes.Add("/subscriptions/$subscriptionId")
+
+        New-AzureRmRoleDefinition -Role $camCustomRoleDefinition | Out-Null
+        $camCustomRoleDefinition = Get-AzureRmRoleDefinition "Cloud Access Manager"
+    }
+
+    # Actions to remove
+    $requiredNotActions = @(
+        # Remove ability to perform any storage accoutn actions
+        "Microsoft.Storage/storageAccounts/*",
+        # Remove ability to get SAS URI of VM Image for Blob access
```

Comment:
Are they still called blobs here? disk access?

Reply:
That's what the documentation called it, "Blob access to the disk"

Reply:
Okay sounds good. Whatever works :)